content/TrueCommand/Clusters.md

```diff
@@ -252,6 +252,7 @@ Click the <i class="material-icons" aria-hidden="true" title="Options">more_vert
 | Setting | Value | Description |
 |---------|-------|-------------|
 | DELETE | button | Remove the share from the Cluster Volume. No data is destroyed in this operation. |
+| CREATE SHARE | button | Opens the form to configure a new SMB share. This allows remote access to this cluster volume. |
```

Comment:
Same question as above concerning the "Value" column and if the new style involves merging this information (button) into the Description.